document_page_procedure/__manifest__.py

```diff
@@ -20,20 +20,22 @@
 ##############################################################################
 {
     "name": "Document Management - Wiki - Procedures",
-    "version": "9.0.1.0.0",
+    "version": "10.0.0",
     "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
     "website": "http://www.savoirfairelinux.com",
     "license": "AGPL-3",
     "category": "Generic Modules/Others",
+    "description": """Procedure Template
+    """,
     "depends": [
-        'document_page_work_instruction'
+        'document_page_work_instructions'
```

Comment:
Hmm... module names are supposed to be singular, not plural.
Are you sure about this change?